src/components/ContactForm/index.jsx

```diff
@@ -32,7 +32,7 @@ export default function ContactForm({ handleShow }) {
           <label htmlFor="fullName">
             {t("contactUs.contactUsForm.name.label")}
             <input
-              className={getErrorClass(errors.fullName)}
+              className={getErrorClass(errors.fullName) || ""}
```

Comment:
you dont want this. the point is you don't want to set a classname if a classname doesnt exist. there is no class, so the snapshot should show no class being set

delete all OR conditions

Reply:
you were right, I thought that `undefined` will be like `null`. done!